openshift/template.yaml

```diff
@@ -60,9 +60,6 @@ objects:
             ports:
               - name: assisted-svc
                 containerPort: 8090
-            envFrom:
```

Comment:
@tsorya aren't you adding live update from the config map?

Reply:
don't have task for it

Reply:
but we started to talk about it, @filanov have an idea where it stopped?

Reply:
@ybettan but why not to have configmap as template?

Reply:
it will block us later when we will add reload of config map

Reply:
We don't need the ConfigMap for 2 reasons:
1. It doesn't add any functionality, those env variables can be defined (template or no template) directly in the deployment - the ConfigMap resource is redundant.
2. k8s doesn't restart a pod on a cm change but it does on a deployment change, so after this PR is merged, if we changes the values of those env parameters (in the deployment) k8s will restart the pod.
Currently if we change the values of the parameters only the cm is updated but the pod need to be restarted manually.


"it will block us later when we will add reload of config map" can you please elaborate on that? what is "adding reload to cm"?
@tsorya

Reply:
it is possible to see if configmap was changed and reload it's values

Reply:
@byount we don't need to restart the pod just because some env changed.

Reply:
Actually if we stop using configmap it will be for the best
When you change any env in the deployment file it will automatically reboot the service. so we will not have to listen to any configmap so maybe it will be a simple solution without any code.